src/client/setup.ts

```diff
@@ -1,6 +1,18 @@
 // ----- Procedures ----- //
 
 import {logger} from "../logger";
+// import { nativeClient } from 'native/nativeApi';
+
+// REPLACE by `nativeClient.getFontSize()`
+function getFontSize() {
```

Comment:
I *think* this might return the following:

```suggestion
function getFontSize(): Promise<number> {
```